core/src/smartcontracts/isi/permissions/combinators.rs

```diff
@@ -21,7 +21,8 @@ impl<O: NeedsPermission, F: IsAllowed<Operation = O>> ValidatorApplyOr<O> for F
 /// *Or*-combinator for two validators
 ///
 /// `check` succeeds if either `first` or `second` validator succeeds
-#[derive(Debug, Clone, Serialize)]
+#[derive(Debug, Clone, Serialize, Display)]
+#[display(fmt = "`{}` or `{}`", first, second)]
```

Comment:
```suggestion
#[display(fmt = "`{}` || `{}`", first, second)]
```

More in line with Java and Rust, less so with Python. This also conveys that the short-circuiting logic is in place.